<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageToolClass</class>
 <widget class="QMainWindow" name="PackageToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PackageTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
      </property>
      <property name="title">
       <string>打包工具</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
              <item>
               <widget class="QPushButton" name="btn_choosePackageSourcePath">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
	font-size:16px;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
                </property>
                <property name="text">
                 <string>选择打包路径</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_sourcePath"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
              <item>
               <widget class="QPushButton" name="btn_choosePackageTargetPath">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
	font-size:16px;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
                </property>
                <property name="text">
                 <string>选择输出路径</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_targetPath"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
           </property>
           <property name="title">
            <string>打包方式</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="topMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QRadioButton" name="rbtn_toZIP">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 14px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       

}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #2196F3;
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 选中时的文字颜色 */
QRadioButton:checked {
    color: #2196F3;
    font-weight: 500;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
                </property>
                <property name="text">
                 <string>输出为ZIP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbtn_toEXE">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 14px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       

}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #2196F3;
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 选中时的文字颜色 */
QRadioButton:checked {
    color: #2196F3;
    font-weight: 500;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
                </property>
                <property name="text">
                 <string>输出为EXE</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
           </property>
           <property name="title">
            <string>打包速度</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="topMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QRadioButton" name="rbtn_fast">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 14px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       

}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #2196F3;
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 选中时的文字颜色 */
QRadioButton:checked {
    color: #2196F3;
    font-weight: 500;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
                </property>
                <property name="text">
                 <string>快速</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbtn_ultra">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    spacing: 8px;
    font-size: 14px;
    color: #333;
    font-weight: bold;        
    color:  rgb(0, 0, 0);       

}

/* 未选中状态 */
QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid #999;
    background: #ffffff;
}

/* 悬停效果 */
QRadioButton::indicator:hover {
    border: 2px solid #666;
}

/* 选中状态 */
QRadioButton::indicator:checked {
    border: 2px solid #2196F3;
}

/* 选中时的内圆 */
QRadioButton::indicator:checked {
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.4,
        fx:0.5, fy:0.5,
        stop:0 #2196F3,
        stop:1 transparent
    );
}

/* 禁用状态 */
QRadioButton::indicator:disabled {
    border: 2px solid #ccc;
    background: #eee;
}

/* 选中时的文字颜色 */
QRadioButton:checked {
    color: #2196F3;
    font-weight: 500;
}

/* 点击动画效果 */
QRadioButton::indicator:pressed {
    border: 2px solid #1565C0;
    background-color: qradialgradient(
        cx:0.5, cy:0.5,
        radius:0.5,
        fx:0.5, fy:0.5,
        stop:0 #BBDEFB,
        stop:1 transparent
    );
}</string>
                </property>
                <property name="text">
                 <string>极致</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_startPackage">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    padding: 6px 14px;
    border: 2px solid #CCC;
    border-radius: 4px;
    background-color: white;
    color: #444;
	font-size:16px;
}

QPushButton:hover {
    border-color: #999;
    background-color: #F5F5F5;
}

QPushButton:pressed {
    border-color: #777;
    background-color: #EEE;
}</string>
                </property>
                <property name="text">
                 <string>开始打包</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="styleSheet">
            <string notr="true">QGroupBox {
    border: 1px solid #DDD;
    border-radius: 4px;
    font: bold 14px;
    color: #666;
    background-color: #F8F8F8;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PackageTool.qrc"/>
 </resources>
 <connections/>
</ui>
